app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomActivity.kt

```diff
@@ -73,6 +76,27 @@ class ChatRoomActivity : AppCompatActivity(), HasSupportFragmentInjector {
         setSupportActionBar(toolbar)
         supportActionBar?.setDisplayShowTitleEnabled(false)
         text_room_name.textContent = chatRoomName
+
+        var drawable: Drawable? = null
+        when (chatRoomType) {
+            RoomType.CHANNEL.toString() -> {
+                drawable = DrawableHelper.getDrawableFromId(R.drawable.ic_room_channel, this)
+            }
+            RoomType.PRIVATE_GROUP.toString() -> {
+                drawable = DrawableHelper.getDrawableFromId(R.drawable.ic_room_lock, this)
+            }
+            RoomType.DIRECT_MESSAGE.toString() -> {
```

Comment:
What if the server settings uses the real name instead of username? We should show the `ic_room_dm` icon only when showing usernames.